### ISSUE TITLE
How to add section without border

### ISSUE THREAD

**Issue by ericchaves** [litixsoft/lx-pdf#20]
Hi folks, is it possible to add a content section without border?

Thanks in advance

**Comment by LitixThomas** [comment]
Of course! Turn of the "debug" option in your template json for Content Sections, or see the documentation for table section.
